src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Query/LaunchProfiles/LaunchProfileActionBase.cs

```diff
@@ -56,7 +56,7 @@ public Task OnRequestProcessFinishedAsync(IQueryProcessRequest request)
 
                     foreach ((IEntityValue projectEntity, EntityIdentity addedProfileId) in group)
                     {
-                        if (handler.RetrieveLaunchProfileEntityAsync(request.QueryExecutionContext, addedProfileId, s_requestedLaunchProfileProperties) is IEntityValue launchProfileEntity)
+                        if (await handler.RetrieveLaunchProfileEntityAsync(request.QueryExecutionContext, addedProfileId, s_requestedLaunchProfileProperties) is IEntityValue launchProfileEntity)
```

Comment:
This is why `{ }` should be used in non-null check pattern matching.

```suggestion
                        if (await handler.RetrieveLaunchProfileEntityAsync(request.QueryExecutionContext, addedProfileId, s_requestedLaunchProfileProperties) is { } launchProfileEntity)
```

Reply:
@sharwell what is this operator called? (is it an operator)?

Reply:
I think it's just called the empty pattern. @jaredpar ?

Reply:
Since when can empty braces be used as a type? How does it know to present itself as `IEntityValue`?

Reply:
> Since when can empty braces be used as a type?

Property patterns were added in C# 8.

> How does it know to present itself as IEntityValue?

That's the result type for the `await` operation. The pattern syntax just adds a null check to it.

Reply:
Changing the patterns we use has implications outside of this one specific example, so that's something for a different change.